docs/sql-guide/functions/function-identifier.md

```diff
@@ -0,0 +1,48 @@
+---
+title: IDENTIFIER()
+layout: default
+parent: Functions
+grand_parent: SQL guide
+---
+
+# IDENTIFIER() function
+
+`IDENTIFIER()` function generates unique IDs for the provided index.
+
+## Syntax
+
+```
+identifier(expr)
+```
+
+## Arguments
+
+| Argument | Description |
+|---|---|
+| `expr` | `index` name as `string`. |
```

Comment:
See comment above regarding `expr`

`index` is technically correct, however the UX and API docs say `table` so we should be standardizing on that.

If the suggestion above is taken into consideration, this would be:

```
| `table_name` | name of existing table | String |
```